multi-node/aws/cmd/kube-aws/command_init.go

```diff
@@ -41,6 +42,7 @@ func runCmdInit(cmd *cobra.Command, args []string) error {
 		{"--region", initOpts.Region},
 		{"--availability-zone", initOpts.AvailabilityZone},
 		{"--key-name", initOpts.KeyName},
+		{"--kms-key-arn", initOpts.KMSKeyARN},
```

Comment:
If this is required, I think we need documentation (at minimum pointing to aws docs) of what an end-user needs to configure prior to using this tool. The ssh-key is well-understood (and required, iirc) to launch new nodes, but setting up kms keys is not as well-understood or required to launch nodes.